tests/support/util.tcl

```diff
@@ -710,6 +710,10 @@ proc generate_fuzzy_traffic_on_key {key duration} {
     set commands [dict create string $string_commands hash $hash_commands zset $zset_commands list $list_commands set $set_commands stream $stream_commands]
 
     set type [r type $key]
+    # Key might have been deleted by active expiration.
+    if {$type eq {none}} {
+        return {}
+    }
```

Comment:
technically, it's not "active" expire (not sure it's "lazy" either). maybe just drop that word.

anyway, i wonder what's the benefit of having expiry set that low when we create the source code.
maybe the better solution is to just change the ttl to be much higher?
i.e. what do we gain or lose by letting the fuzzer work on ttls that are in the past?

alternatively, maybe we can add one non-volatile field to that hash key.
again do we lose anything by doing that?

@moticless @tezc WDYT?

Reply:
Potentially, it can expose a bug that is only visible when we expire keys. e.g. restore is successful (we think so), later on expiration, it triggers a crash/assert or accesses a dangling pointer...

If we can get away with the above if check, maybe we should just leave as it is.

Reply:
ok, i'm a little afraid that this skip is too broad and can hide other bugs.
maybe we can pass in the TYPE from the caller, and let the caller realize the RESTORE deleted the key?

Reply:
Also we know if the key is hash with volatile fields (key name contains with hashmd). We may check key's name to understand if expiry is expected.

Reply:
> Potentially, it can expose a bug that is only visible when we expire keys. e.g. restore is successful (we think so), later on expiration, it triggers a crash/assert or accesses a dangling pointer...

yes, this is the reason why i don't use `debug set-active-expire 0` to avoid the key to be expired.

Reply:
> alternatively, maybe we can add one non-volatile field to that hash key. again do we lose anything by doing that?

@oranagra even so, the non-volatile field can also be changed to volatile field by `corrupt_payload()`.

Reply:
> Also we know if the key is hash with volatile fields (key name contains with hashmd). We may check key's name to understand if expiry is expected.

i'm not a fan of it, it means that the caller need to know the details of `generate_types()`.